# This expresses velocity in free space broken into its linear and angular parts.

float32 odom_linear
float32 odom_angular

int8 camera_angle

float32 linear_module
float32 putter_1
float32 putter_2

int8 oblique_angle
int8 oblique_drawer
int8 flat_drawer
uint8 belt
int8 camera_tilt